pipeline_manager/src/config.rs

```diff
@@ -18,28 +19,44 @@ fn default_working_directory() -> String {
     ".".to_string()
 }
 
-/// Pipeline manager configuration read from a YAML config file.
-#[derive(Deserialize, Clone)]
+fn default_sql_compiler_home() -> String {
+    "../sql-to-dbsp-compiler".to_string()
```

Comment:
PS. We are moving to the directory layout where the SQL compiler lives in the main DBSP folder (even if it's a separate repo)